src/worker/worker/collectors/base_web_collector.py

```diff
@@ -59,3 +66,51 @@ def xpath_extraction(self, html_content, xpath: str) -> str:
             logger.error(f"No content found for XPath: {xpath}")
             return ""
         return document.xpath(xpath)[0].text_content()
+
+    def extract_meta(self, web_content, web_url):
+        # Trafilatura.extract_metadata() raises for certain URLs a ValueError
+        # Example URL: https://www.mozilla.org/en-US/security/advisories/mfsa2024-17/
+        # Example error message: "month must be in 1..12"
+        metadata = extract_metadata(web_content, default_url=web_url)
+        author = "" if metadata is None or metadata.as_dict().get("author") is None else metadata.as_dict().get("author")
+        title = "Title not collected" if metadata is None or metadata.as_dict().get("title") is None else metadata.as_dict().get("title")
+
+        return author, title
```

Comment:
```
from contextlib import suppress

with suppress(ValueError):
  metadata = extract_metadata(web_content, default_url=web_url)
 
 if not metadata:
   return "", ""
  
  return metadata.as_dict().get("author", ""), metadata.as_dict().get("title", "")
  ```

Reply:
This is not the best way, because it can still return None, None. Default value "" is returned only if the key does not exist at all.